src/org/opensolaris/opengrok/logger/LoggerUtil.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2008, 2010, Oracle and/or its affiliates. All rights reserved.
+ */
+package org.opensolaris.opengrok.logger;
+
+import org.opensolaris.opengrok.logger.formatter.ConsoleFormatter;
+import org.opensolaris.opengrok.logger.formatter.FileLogFormatter;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.logging.*;
+
+/**
+ * Utilities to maintain logging.
+ */
+public class LoggerUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoggerUtil.class);
+
+    public static final String BASE_LOGGER = "org.opensolaris.opengrok";
+    private static final int DEFAULT_FILEHANDLER_LIMIT = 52428800;
+    private static final int DEFAULT_FILEHANDLER_COUNT = 3;
```

Comment:
I'd love to get as much stuff as possible out of static classes and put into logger.properties ...